### ISSUE TITLE
Optimise vocab loading

### ISSUE THREAD

**Issue by wu-lee** [SolidarityEconomyAssociation/sea-map#133]
Generating the vocabs gets very slow when ther are more than one, delaying the map loading for 30 seconds or longer.  This can probably be sped up with more selective loading and/or caching.

**Comment by wu-lee** [comment]
It turns out this is just that vocab queries don't have the default graph uri set.

I compared the speed of `get_vocabs.php` to a command-line SPARQL query via `curl` (see [seodo](https://gist.github.com/wu-lee/0e02d21503db95d534bfb12fcd1d20e3) using the same query, and the latter was less than half a second. The critical difference was the `default-graph-uri`.

This info also is in the `config/*/default-graph-uri.txt` files of a map website, so in principle `get_vocabs.php` could get it from there, but there vocab queries are done once and currently shared by all datasets, so it isn't clear how to resolve that and pick the right one.

Therefore I decided it was a parameter like `endpoint` in `config.json`, which needs to be provided once per vocab query. (I've left it optional however, so existing sea-map sites will still work without it.)
